pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/visitor/JavaCodeWriter.java

```diff
@@ -785,7 +786,7 @@ public CodeToResolve visitExprLike(ExprLike node) {
 
     private static String[] tmplExprBuiltinFuncCall =
             new String[] {
```

Comment:
```suggestion
          {
```
new String[] 는 없어도 될 거 같습니다.

Reply:
오... 그렇군요. 몰랐네요.
이렇게 한 부분이 워낙 많아서 변경 시 리뷰에 혼란을 줄 수 있으니 
리뷰가 완료 된 후 일괄 적용 하겠습니다. 
감사합니다.